frontend/e2e/nightwatch-entrypoint.bash

```diff
@@ -5,7 +5,25 @@ function trapint {
     exit 0
 }
 
-echo 'Waiting for app to boot...'
+echo 'Waiting for backend to boot...'
+curl -k http://localhost:8080/health > /dev/null 2>&1
+result=$?
+
+polls=0
+while [[ $result -ne 0 && $polls -lt 120 ]]; do
```

Comment:
Ten minutes is a long time to wait for startup of an app that only typically takes 15-30 seconds to start—I feel like this was originally a 2-minute wait and then the sleep got changed from 1 second to 5?

Reply:
Agreed, this definitely doesn't need to be as long